poolmgr/gpm.go

```diff
@@ -157,8 +157,8 @@ func (gpm *GenericPoolManager) eagerPoolCreator() {
 		// creating pools for envs that are actually used by functions.  Also we might want
 		// to keep these eagerly created pools smaller than the ones created when there are
 		// actual function calls.
-		for _, env := range envs {
-			_, err := gpm.GetPool(&env)
```

Comment:
If GenericPool store the value of the `env`, then its find. 
 
```
GenericPool struct {
    env              *fission.Environment
    ...
}
```
But since GenericPool store the address of the `env`, all generic pools contain the same address of the loop variable which's value is the last element of the env slice. 
Its the same issue with PR https://github.com/fission/fission/pull/141